docs/concepts/vega-chain.md

```diff
@@ -143,37 +143,113 @@ Read more: [Risks of over-staked validators](/docs/concepts/vega-chain#too-much-
 
-In this version of restricted mainnet, validators will not lose stake or rewards if they have a temporary interruption of service.
+## Consensus validating nodes
+Consensus validating nodes are responsible for agreeing on the order of transactions and creating new blocks so that all nodes can agree on the state of the network.
 
-### Validator node performance
-A validator node's performance is expressed through a validator score, and in the future, a performance score as well. 
+They receive rewards based on having enough (but not too much) self-stake, as well as how many tokenholders nominate them. The tokenholders who nominate them also receive a cut of the rewards.
 
-The validator score is calculated for each epoch, based on how much stake a validator has as well as other factors including the total number of validators and the optimal stake. 
+If a consensus validator's stake or performance is subpar, their validator score will be lowered, and that validator's node will be chosen less frequently to propose a block, because Vega feeds the voting power of each validating node to the Tendermint consensus algorithm.
 
-If, at the end of an epoch, a validator does not have sufficient stake self-nominated or has overall too much stake, then their validator score will be lowered, which can impact the rewards a validator and its nominators receive. 
+This can also affect the rewards they and their nominators receive.
+If a consensus validating node stops validating, or performs poorly, then a standby validator can replace it.
+
+### How consensus validators are chosen
+Consensus validators are chosen based on a range of variables, including the validator scores, and the number of available slots. If there are no empty slots, at most one validator can be changed per epoch.
+
+If a standby validator has a better overall performance than an existing consensus validator, then a standby validator can be promoted to replace a consensus validator.
+
+## Standby validators
+Standby (also called ersatz) validators do not contribute to the chain, but are set up to join the consensus validator set if an existing validator drops off. As they don’t participate in consensus, they don’t need to be registered with a multisig contract.
+
+As with the consensus validators, standby validators are defined based on how much self-stake and nominated stake they have.
+
+Standby validators, and the tokenholders who stake them, receive a share of rewards. The rewards for standby validators are calculated and penalised in the same way as consensus validators, except scaled down. How much they are scaled is based on the network parameter `network.validators.ersatz.rewardFactor`.
+
+:::info 
+The network is set to not allow any standby validators for alpha mainnet, and the number of validators will be increased via governance as early alpha mainnet progresses.
+:::
+
+### How standby validators are chosen
```

Comment:
I think it's worth clarifying (as you're using the term standby) that there are 3 classes, consensus, standby, and pending. Where consensus and standby are explained and are compensated, but pending are on the waiting list to be accepted as standby (hence being compensated)